src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutRecoverabilityBehavior.cs

```diff
@@ -7,6 +7,7 @@ namespace NServiceBus
     using System;
     using System.Collections.Generic;
     using System.Threading.Tasks;
+    using JetBrains.Annotations;
```

Comment:
Let the derailing continue...

I'm not a fan of hints for static analysis tools (R# or otherwise).

Also:
> We all use R# so it might benefit us when writing code

I think there are a few that don't use R#. Some prefer CodeRush, some vanilla, some OmniSharp.

I think it's a good idea to not rely on R# features in the long term, but don't think there's any major value in derailing this work just because of that.